l10n_it_fatturapa_pec/views/fatturapa_attachment_out.xml

```diff
@@ -6,11 +6,11 @@
         <field name="model">fatturapa.attachment.out</field>
         <field name="inherit_id" ref="l10n_it_fatturapa_out.view_fatturapa_out_attachment_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//div/group[last()]" position="after">
+            <group name="preview" position="after">
                 <group name="last_sdi_response" string="Last Response from Exchange System" attrs="{'invisible':[('state', '=', 'ready')]}">
                     <field name="last_sdi_response" nolabel="1"/>
                 </group>
-            </xpath>
+            </group>
```

Comment:
Questo commit non potrebbe essere incluso in 9d795428309c2d61b67bf79847ce2a659f24c210 ?

Reply:
Non hanno scopi poi così diversi, fatto